scripts/searchSamadi.py

```diff
@@ -0,0 +1,20 @@
+import commands, sys
+
+stringToLookFor = "*v0.1.0+76X_HHAnalysis_2016-04-04.v0"
```

Comment:
If you prefer I can but it does not change every week. Also I like to have this string hardcoded there so I can launch it without having to look for the last tag everytime.